.vib/opencart/cypress/cypress/integration/opencart_spec.js

```diff
@@ -0,0 +1,114 @@
+/// <reference types="cypress" />
+import { random } from './utils';
+
+it('allows a user to register', () => {
+  cy.visit('/index.php?route=account/login');
+  cy.contains('Continue').click();
+  cy.fixture('users').then((user) => {
+    cy.get('#input-firstname').type(`${user.newUser.firstName}.${random}`);
+    cy.get('#input-lastname').type(`${user.newUser.lastName}.${random}`);
+    cy.get('#input-email').type(`${random}.${user.newUser.email}`);
+    cy.get('#input-telephone').type(user.newUser.phone);
+    cy.get('#input-password').type(`${user.newUser.password}.${random}`);
+    cy.get('#input-confirm').type(`${user.newUser.password}.${random}`);
+  });
+  cy.get('[type="checkbox"]').check();
+  cy.contains('Continue').click();
+  cy.contains('Your Account Has Been Created!');
+});
+
+it('allows a user to place an order', () => {
```

Comment:
We should add to the description of the test something like "and an admin to list it". 

The last part where an admin login wouldn't be necessary to verify an order was made as we are already doing that in L50. In that case, the admin check can be seen as testing the listing of orders, wdyt?

Reply:
I changed the name of the test, good catch!

However, I disagree about the listing of order tests. IMHO this is a sort of integration test between the admin component and the shop.
First I place an order, and then I check if I placed the order successfully _in the shop_.
Then if this is successful in the shop, I log in as admin and check if the order is visible in the admin component.

Reply:
I see your point and it is indeed a more real use-case. LGTM!